src/main/java/io/github/fixitlater/quizapi/question/QuestionService.java

```diff
@@ -0,0 +1,69 @@
+package io.github.fixitlater.quizapi.question;
+
+import io.github.fixitlater.quizapi.Category;
+import io.github.fixitlater.quizapi.Language;
+import lombok.AllArgsConstructor;
+import org.apache.commons.math3.random.RandomDataGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
Polecałbym unikanie tych konstrukcji z częścią `onConstructor = @__(@Autowired)` jest mało czytelne i w praktyce unika się stosowania tej konstrukcji.
Po drugie nowy Spring w nowszych wersjach powinien poradzić sobie z wstrzykiwaniem bez `@Autowired` 

Sami twórcy stwierdzili również, że jest to eksperymentalny ficzer i aby używać tego ostrożnie:
```
To put annotations on the generated constructor, you can use onConstructor=@__({@AnnotationsHere}), but be careful; this is an experimental feature. For more details see the documentation on the onX feature.
```